src/store/configureStore.js

```diff
@@ -4,43 +4,36 @@ import thunk from "redux-thunk";
 import reduxImmutableStateInvariant from "redux-immutable-state-invariant";
 import createHistory from "history/createBrowserHistory";
 import { routerMiddleware } from "connected-react-router";
+import { persistStore, persistReducer } from "redux-persist";
+import storage from "redux-persist/lib/storage"; // defaults to localStorage for web
 
 import createRootReducer from "../reducers";
 
 export const history = createHistory();
 
 function configureStoreDev(initialState) {
   const reactRouterMiddleware = routerMiddleware(history);
-  const middleWare = [
-    reduxImmutableStateInvariant(),
-    thunk,
-    logger,
-    reactRouterMiddleware
-  ];
-
-  return createStore(
-    createRootReducer(history),
-    initialState,
-    applyMiddleware(...middleWare)
-  );
+  const middleWare = [reduxImmutableStateInvariant(), thunk, logger, reactRouterMiddleware];
+
+  const persistConfig = {
+    key: "root",
+    storage
+  };
+
+  const persistedReducer = persistReducer(persistConfig, createRootReducer(history));
+  const store = createStore(persistedReducer, initialState, applyMiddleware(...middleWare));
+  const persistor = persistStore(store);
+
+  return { store, persistor };
 }
 
 function configureStoreProd(initialState) {
   const reactRouterMiddleware = routerMiddleware(history);
-  const middleWare = [
-    thunk,
-    reactRouterMiddleware
-  ];
-
-  return createStore(
-    createRootReducer(history),
-    initialState,
-    applyMiddleware(...middleWare)
-  );
+  const middleWare = [thunk, reactRouterMiddleware];
+
+  return createStore(createRootReducer(history), initialState, applyMiddleware(...middleWare));
 }
 
-const configureStore = process.env.NODE_ENV === "production"
-  ? configureStoreProd
-  : configureStoreDev;
+const configureStore = process.env.NODE_ENV === "production" ? configureStoreProd : configureStoreDev;
```

Comment:
@davishooly  you have persisted the data in development configuration but you have not persisted in production configuration. Kindly add persist configuration in `configureStoreProd ` function.

Reply:
👍